Quorum/tests/test_checks.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+import Quorum.tests.utils as utils
+
+import Quorum.checks as Checks
+from Quorum.utils.chain_enum import Chain
+from Quorum.apis.price_feeds import ChainLinkAPI
+import Quorum.config as config
+
+from pathlib import Path
+import shutil
+import json
+
+from typing import Generator
+
+
+@pytest.fixture
+def tmp_output_path() -> Generator[Path, None, None]:
+    og_path = config.MAIN_PATH
+    config.MAIN_PATH = Path(__file__).parent / 'tmp'
+    yield config.MAIN_PATH  # Provide the temporary path to the test
+    shutil.rmtree(config.MAIN_PATH)
+    config.MAIN_PATH = og_path
```

Comment:
This should be exported to a conftest.py file (common practice)

Reply:
But isn't conftest used for every test?